load_blazegraph_test.sh

```diff
@@ -10,16 +10,85 @@ BASEURI="http://ld4p-test.stanford.edu/"
 # vars above this line may need to change to process other data
 #------------------------------------------------
 
-BG_URL="http://localhost/blazegraph/"
-TEST_NAMESPACE=test
```

Comment:
The `kb` namespace is always available.  If the `test` namespace is not manually created, this test script will fail, but that's not a failure of the loader.  For this test script to be robust and reliable, using the `kb` namespace is better.